src/test/java/br/com/caelum/mog/services/DownloadsServiceTest.java

```diff
@@ -1,14 +1,17 @@
 package br.com.caelum.mog.services;
 
-import br.com.caelum.mog.adapters.Downloadable;
-import br.com.caelum.mog.domains.models.Course;
-import br.com.caelum.mog.domains.models.Customer;
-import br.com.caelum.mog.domains.models.Offer;
-import br.com.caelum.mog.domains.models.CaelumInfo;
-import br.com.caelum.mog.domains.models.CareOf;
-import br.com.caelum.mog.domains.models.Responsible;
-import br.com.caelum.mog.domains.models.CourseSummaryItem;
-import br.com.caelum.mog.enums.CaelumDistrict;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.Matchers.hasKey;
+import static org.junit.Assert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import java.time.LocalDate;
+import java.time.Month;
+import java.util.Arrays;
+import java.util.Arrays.*;
```

Comment:
[Minor do minor] Caso você for arrumar alguma coisa nesse projeto, poderia tirar esse import que não esta sendo utilizado?

Reply:
Claro :rocket: